documentation/modules/operators/ref-operator-cluster-feature-gates.adoc

```diff
@@ -74,7 +74,7 @@ Examples of the `KafkaNodePool` resources can be found in the xref:config-exampl
 
 .Downgrading from KafkaNodePools
 
-If your cluster already uses `KafkaNodePool` custom resources, and you wish to downgrade to an older version of Strimzi that does not support them or with the `KafkaNodePools` feature gate disabled, you must first migrate from `KafkaNodePool` custom resources to managing Kafka nodes using only `Kafka` custom resources. For more information, see the instructions for xref:proc-migrating-clusters-node-pools-str[reversing a migration to node pools].
+KRaft.based Apache Kafka cluster always use KafkaNodePools and downgrading from them is not possible anymore.
```

Comment:
This might be interpreted as downgrading is not possible.  Maybe something like:

> KRaft-based Kafka clusters always use `KafkaNodePools`. 
> Downgrading to a cluster without node pools is not possible.